libraries/ms-common/inc/pqueue.h

```diff
@@ -21,7 +21,7 @@ typedef struct PQueue {
 void pqueue_init(PQueue *queue, PQueueNode *nodes, size_t num_nodes);
 
 // Push a node with the specified priority and data onto the queue.
-StatusCode pqueue_push(PQueue *queue, void *data, uint16_t prio);
+StatusCode pqueue_push(PQueue *queue, void *data, uint32_t prio);
```

Comment:
Originally wanted a pqueue for timers concluded it would have to be entirely or partly rebuilt every time any timer got updated.